lighthouse-plugin-publisher-ads/audits/duplicate-tags.js

```diff
@@ -41,7 +43,7 @@ const tags = [
  * @type {LH.Audit.Details.Table['headings']}
  */
 const HEADINGS = [
-  {key: 'url', itemType: 'url', text: headings.url},
+  {key: 'script', itemType: 'url', text: headings.url},
```

Comment:
Could you please change the last part to headings.script, and likewise change the referenced key in the JSON file?